spec/requests/trip_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'Trip index', type: :request do
+  let(:json_response) { JSON.parse(response.body) }
+
+  it 'has a valid factory' do
+    expect(FactoryBot.create(:trip)).to be_valid
+  end
+
+  let(:get_trips) { get '/trips' }
```

Comment:
I suggest naming this by using rspec [explicit subject](https://relishapp.com/rspec/rspec-core/v/3-8/docs/subject/explicit-subject)